stack.yaml

```diff
@@ -3,3 +3,4 @@ resolver: nightly-2017-01-25
 packages:
   - "."
   - "./docs/source/tutorial"
+  - "./graphql-wai"
```

Comment:
How would a mono-repo be different? Move top-level stuff to sub-dirs? OK with that but year, separate PR